analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -42,7 +42,7 @@ import org.ossreviewtoolkit.utils.core.log
 /**
  * The class to run the analysis. The signatures of public functions in this class define the library API.
  */
-class Analyzer(private val config: AnalyzerConfiguration) {
+class Analyzer(private val config: AnalyzerConfiguration, private val labels: Map<String, String>) {
```

Comment:
Sorry, one more thing: Should we default this to `= emptyMap()` like we do in `OrtResult` and `Defect` already, to avoid some of the trivial code changes?